socialdistribution/profiles/views.py

```diff
@@ -99,6 +99,25 @@ def edit_profile(request):
 
     return render(request, template, context)
 
+def view_author_profile(request, author_id):
+    #The user who login in/use the application
+    # TODO: add cookie or token to store the user
+    user_author = request.user
+    
+    author = Author.objects.get(id=author_id)
+    template = 'profiles/profiles_view.html'
+    # form = ProfileForm(instance=author)
+    status = True
+
+    if author == user_author:
```

Comment:
You could just redirect to `/viewprofile` if `author == user_author` instead duplicating the template code using the status flag. Or scrap the `viewprofile` and just use this.

Reply:
I think, even if the user itself, it can be see in these url

Reply:
Sure, but it just causes inconsistency. There are two urls that handle the same functionality just differently depending on status.